pages/events/[slug].vue

```diff
@@ -41,216 +44,239 @@ watch(data, (newVal, oldVal) => {
   console.log('In watch preview enabled, newVal, oldVal', newVal, oldVal)
   page.value = newVal
```

Comment:
previews wont work, we need to add the following code to this watch method
` page.value = _get(data.value, 'ftvaEvent', {})
 series.value = _get(data.value, 'ftvaEventSeries', {}) `